e2e/suite1/connections-actions.test.ts

```diff
@@ -0,0 +1,295 @@
+import * as connections from '../helpers/connection-actions';
+import disableTransitions from '../helpers/disableTransitions';
+import insertFixtures from '../helpers/insertFixtures';
+import { adminLogin, logout } from '../helpers/login';
+import proxyMock from '../helpers/proxyMock';
+
+describe('connections', () => {
+  beforeAll(async () => {
+    await insertFixtures();
+    await proxyMock();
+    await adminLogin();
+    await disableTransitions();
+  });
+
+  afterAll(async () => {
+    await logout();
+  });
+
+  it('should navigate to an entities connections view', async () => {
+    await expect(page).toClick(
+      '#filtersForm > div:nth-child(2) > ul > li > ul > li:nth-child(6) > label > span.multiselectItem-name > span'
+    );
+    await expect(page).toClick('.item-name', {
+      text: 'Acevedo Buendia et al (Discharged and Retired Employees of the Office of the Comptroller)',
+    });
+    await expect(page).toClick(
+      '#app > div.content > div > div > div > aside.side-panel.metadata-sidepanel.is-active > div.sidepanel-footer > span > a'
```

Comment:
I thought we moved away from this type of selectors that can break easily with any sort of change to the code or even the class names?  Instead we now use selectors like "button with text `save`", or things like that?  Also, when using these type of selectors in the past we have created a lookup table and use more semantic names so that people reading the test have an idea of what it is that the test is supposed to be clicking or selecting.

Reply:
Simplified the selectors.